Format generated test file

webpro opened the pull request:
See https://github.com/kadena-community/kadena.js/pull/291#discussion_r1211732301

This is a quick fix so the generated file is reformatted according to our Prettier config. Ideally we would do it directly in code but that would require more changes and adding the dependency, but this is a test project so I didn't bother.

It's kinda weird that we have to take into account that Rush does not execute `npm run pre[task]` or `npm run post[task]` but in this case we can take advantage of it.

alber70g commented:
I thought we removed it from being formatted and added it to `.prettierignore`. Maybe an idea to not format generated code in general?

webpro commented:
> I thought we removed it from being formatted and added it to `.prettierignore`. Maybe an idea to not format generated code in general?

Yes, totally agree. I assumed that the problem had to do with prettier not picking up the root ignore list when executed from a package dir, or something else more complicated (since nobody fixed it). But everything seems OK afaic, so I just pushed a change so this PR results in just an unformatted generated file.
